js/script.js

```diff
@@ -631,6 +631,13 @@ var ot_validCombos = {
     
     "ot_android,ot_on_device,ot_java,ot_X64,ot_CPU":
     "Add a dependency on <a href='https://mvnrepository.com/artifact/com.microsoft.onnxruntime/onnxruntime-training-android' target='_blank'>com.microsoft.onnxruntime:onnxruntime-training-android</a> using Maven/Gradle and refer to the instructions <a href='https://onnxruntime.ai/docs/install/#install-for-on-device-training' target='_blank'>here.</a>",
+
+    "ot_mac,ot_on_device,ot_python,ot_X64,ot_CPU":
+    "pip install onnxruntime-training<br/><br/>*</b><a href='https://download.onnxruntime.ai/' target='blank'>Available versions</a>",
+
+    "ot_ios,ot_on_device,ot_objc,ot_X64,ot_CPU":
+        "Add 'onnxruntime-training-c' or 'onnxruntime-training-objc' using CocoaPods and refer to the <a href='https://onnxruntime.ai/docs/tutorials/mobile/' target='_blank'>mobile deployment guide</a>",
```

Comment:
I think these instructions can also be enabled for C and C++ APIs

Reply:
for iOS platform?

Reply:
> for iOS platform?

yep. actually, `onnxruntime-training-objc` for Objective-C and `onnxruntime-training-c` for C/C++ would match the inference package instructions.

![image](https://github.com/microsoft/onnxruntime/assets/18449977/00c949d0-81f7-421f-a318-ea7a42b4f287)
![image](https://github.com/microsoft/onnxruntime/assets/18449977/00d0b5ac-5696-4e6a-8d6b-c7e10b973d38)